eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "rallycoding"
```

Comment:
`react-app` may be?

Reply:
It can be any dictionary to varify I have been using  rallycoding for my projects